src/formatter/date.ts

```diff
@@ -0,0 +1,95 @@
+import {
+  DateFromString,
+  DateFromTimestamp,
+  UtcOnlyIsoDateFromString
+} from "@pagopa/ts-commons/lib/dates";
+import * as E from "fp-ts/Either";
+import { identity, pipe } from "fp-ts/function";
+import * as t from "io-ts";
+import { failure } from "io-ts/PathReporter";
+
+const onLeft = (errors: t.Errors): string =>
+  pipe(failure(errors), e => e.join("\n"));
+
+export const dateStringToUtcFormat = (dateString: string): string =>
+  pipe(
+    dateString,
+    DateFromString.decode,
+    E.match(onLeft, date => date.toUTCString())
+  );
+
+export const dateStringToDateObject = (dateString: string): string | Date =>
```

Comment:
If this method is used just in the tests, maybe is better to move it in the test file and remove the export.